src/main/java/com/github/jreddit/entity/User.java

```diff
@@ -107,4 +111,56 @@ public void connect() throws IOException, ParseException {
         return values;
     }
 
+
+    /**
+     * This function returns all the subreddits the user is subscribed to.
+     * @return A list of subreddit objects
+     * @throws RetrievalFailedException    If retrieval of subreddits fails
+     * @throws RedditError
+     */
+    public List<Subreddit> getSubscribed() throws RetrievalFailedException, RedditError {
+        if (this.getCookie() == null || this.getModhash() == null) {
+            System.err.printf("Please invoke the connect method in order to login the user");
+            return null;
+        }
+        Subreddits sub = new Subreddits(restClient, this);
+
+        // Automatically returns maximum allowed number of objects, should this be a parameter?
+        return sub.parse(ApiEndpointUtils.USER_GET_SUBSCRIBED + "?&limit=100");
+    }
+
+    /**
+     * This function returns all the subreddits the user is an approved contributor to.
+     * @return A list of subreddit objects
+     * @throws RetrievalFailedException    If retrieval of subreddits fails
+     * @throws RedditError
+     */
+    public List<Subreddit> getContributedTo() throws RetrievalFailedException, RedditError {
+        if (this.getCookie() == null || this.getModhash() == null) {
+            System.err.printf("Please invoke the connect method in order to login the user");
+            return null;
+        }
+        Subreddits sub = new Subreddits(restClient, this);
+
+        // Automatically returns maximum allowed number of objects, should this be a parameter?
+        return sub.parse(ApiEndpointUtils.USER_GET_CONTRIBUTED_TO + "?&limit=100");
+    }
+
+    /**
+     * This function returns all the subreddits the user is a moderator of.
+     * @return A list of subreddit objects
+     * @throws RetrievalFailedException    If retrieval of subreddits fails
+     * @throws RedditError
+     */
+    public List<Subreddit> getModeratorOf() throws RetrievalFailedException, RedditError {
+        if (this.getCookie() == null || this.getModhash() == null) {
+            System.err.printf("Please invoke the connect method in order to login the user");
+            return null;
+        }
+        Subreddits sub = new Subreddits(restClient, this);
+
+        // Automatically returns maximum allowed number of objects, should this be a parameter?
+        return sub.parse(ApiEndpointUtils.USER_GET_MODERATOR_OF + "?&limit=100");
```

Comment:
I am not sure about this. I'm thinking maybe leave the limit=100 as default, and have the possibility to add limit if needed. i.e. if limit == null { limit = 100 }